room/compiler-processing/src/main/java/androidx/room/compiler/processing/XFiler.kt

```diff
@@ -23,11 +23,22 @@ import com.squareup.kotlinpoet.FileSpec
  * Code generation interface for XProcessing.
  */
 interface XFiler {
-    fun write(javaFile: JavaFile)
+    /**
+     * @param aggregating Specifies whether this
```

Comment:
lets make this an enum (can still have the default).
Boolean makes it harder to read if not called w/ named parameters.